src/modules/visitor-space/components/AddToFolderBlade/AddToFolderBlade.tsx

```diff
@@ -27,6 +27,7 @@ const AddToFolderBlade: FC<AddToFolderBladeProps> = (props) => {
 	const { tHtml } = useTranslation();
 	const { onSubmit, selected } = props;
 	const [pairs, setPairs] = useState<AddToFolderFormStatePair[]>([]);
+	const [lastCreatedFolderId, setLastCreatedFolderId] = useState('');
```

Comment:
prefer an actual null value for something being empty
```suggestion
	const [lastCreatedFolderId, setLastCreatedFolderId] = useState<string | null>(null);
```